primitives/src/rescue/mod.rs

```diff
@@ -161,8 +163,7 @@ impl<F: PrimeField> RescueVector<F> {
         aux
     }
 
-    fn add_assign_elems(&mut self, elems: &[F]) {
-        assert_eq!(elems.len(), STATE_SIZE);
+    fn add_assign_elems(&mut self, elems: &[F], _expected_size: usize) {
```

Comment:
Any reason to add a new but unused parameter here?

Reply:
That was a leftover from experimentation. Previously we only allowed the number of elements to have a certain size. Now the `absorb` method will take any number of elements, and the length check is enforced by the `_with_padding` methods of the structs using `absorb`, if desired.
Done in https://github.com/EspressoSystems/jellyfish/pull/146/commits/c094a0d0acaed02e2b0ce602006f48ab02e0246d.